desktop/cmp/chart/plugin/zoomout.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable consistent-this */
```

Comment:
Couple of questions:

+ Any comment on where this comes from?  Why do we need the chart variable instead of this?

Reply:
I've removed the chart variable the eslint-disable.  The code works fine, but I think the feeling was that it was confusing to track when reading the several event handler methods that 'this' always refers to the chart.